examples/fibonacci.cpp

```diff
@@ -0,0 +1,49 @@
+#include <array>
+#include <cassert>
+#include <iostream>
+
+#include <beman/inplace_vector/inplace_vector.hpp>
+
+using namespace beman::inplace_vector;
+
+/**
+ * Generates fibonacci sequence using inplace_vector.
+ * See: https://en.wikipedia.org/wiki/Fibonacci_sequence
+ */
+template <int Capacity> inplace_vector<int, Capacity> fibonacci_to(int num) {
+  assert(num < Capacity);
+
+  inplace_vector<int, Capacity> vec;
+
+  for (auto i = 0; i <= num; ++i) {
+    static std::array<int, 2> first_two = {0, 1};
```

Comment:
Maybe move this static variable outside the loop? I don't see a reason why it needs to be declared inside the loop. It could also be made `constexpr`.

Reply:
Done.